src/jbi/router.py

```diff
@@ -1,34 +1,79 @@
 """
 Router dedicated to Jira Bugzilla Integration APIs
 """
+import importlib
 import logging
-from typing import Optional
+from types import ModuleType
+from typing import Dict, List, Optional
 
-from fastapi import APIRouter, Depends, Request
-from fastapi.responses import HTMLResponse
+from fastapi import APIRouter, Body, Depends, Request
+from fastapi.responses import HTMLResponse, JSONResponse
 from fastapi.templating import Jinja2Templates
 
 from src.app import environment
 from src.jbi import configuration
+from src.jbi.bugzilla_objects import BugzillaBug, BugzillaWebhookRequest
+from src.jbi.model import ActionError, IgnoreInvalidRequestError
```

Comment:
Shouldn't `model` be plural? `models`? And shouldn't errors go to their own `errors` module?